plugins/chain_plugin/test/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ add_executable( test_chain_plugin
         test_account_query_db.cpp
         test_trx_retry_db.cpp
         test_trx_finality_status_processing.cpp
+        test_chainbase_types.cpp
```

Comment:
I think this test should be in `unittests` not in `chain_plugin` tests.

Reply:
moved.